app/views/coronavirus_form/results.html.erb

```diff
@@ -17,12 +17,14 @@
 <% if result_groups(session).empty? %>
   <%= sanitize(t("coronavirus_form.results.no_results")) %>
 <% else %>
-  <% result_groups(session).each do |group| %>
-    <%= render "components/actions-group", {
-      title: group[1][:heading],
-      subsections: group[1][:questions]
-    } %>
-  <% end %>
+  <div data-track-ec-list="Find coronavirus support">
```

Comment:
Non-blocker. If time allows it would be good to avoid this extra div and pass the data attribute to the action-group component; as a suggestion, calling it with `data_attributes: { "track-ec-list": "Find coronavirus support" }` and updating the component to have [`data_attributes` in the section element](https://github.com/alphagov/govuk-coronavirus-find-support/blob/master/app/views/components/_actions-group.html.erb#L5).

Reply:
We only want the `data-track-ec-list` attribute to be added to the one element wrapping all of the results so the position of each result can be reported.

Because the action-group component is called multiple times on the results page this would cause `data-track-ec-list` to be added to multiple sections on the page, rather than once. 

This would mean that the position of each result would be reported in relation to each section, rather than in relation to the entire page.